nislmigrate/migrators/file_migrator.py

```diff
@@ -124,7 +143,9 @@ def pre_capture_check(
             arguments,
             self.config(facade_factory)
         )
-
+        captured_file_store_root_path = os.path.join(migration_directory, _SAVED_OLD_FILE_STORE_ROOT_FILE_NAME)
+        if configuration.file_facade.does_file_exist(captured_file_store_root_path):
+            raise MigrationError(_CLEAR_MIGRATION_DIRECTORY_BEFORE_CAPTURING_ERROR)
```

Comment:
Is this in the wrong place? This is restore, not capture.

Reply:
This was in the correct place since that file is written during capture, and this was in the pre_capture_check method, but I've now removed it since it's fine for that file to just be overwritten during capture.